.eslintrc.js

```diff
@@ -0,0 +1,58 @@
+module.exports = {
+	root: true,
+	plugins: ["react", "prettier", "@typescript-eslint"],
+	env: {
+		browser: true,
+	},
+	parser: "@typescript-eslint/parser",
+	parserOptions: {
+		ecmaVersion: 6,
+		sourceType: "module",
+		ecmaFeatures: {
+			jsx: true,
```

Comment:
Question - should this be `tsx`?

Reply:
no, as the parser options object that eslint expects does not have a `tsx` key. however, that should not impact the performance of eslint. read more here: https://eslint.org/docs/user-guide/configuring/language-options#specifying-parser-options

Reply:
Thank you for explaining!